pkg/server/http/http.go

```diff
@@ -17,6 +15,11 @@ import (
 	toolkit_web "github.com/prometheus/exporter-toolkit/web"
 	"golang.org/x/net/http2"
 	"golang.org/x/net/http2/h2c"
+	"net/http"
```

Comment:
please fix the import convention for go: https://google.github.io/styleguide/go/best-practices.html#import-ordering

Reply:
refactored